napari/utils/_tests/test_dtype.py

```diff
@@ -2,13 +2,14 @@
 
 import numpy as np
 import pytest
-import tensorstore as ts
-import torch
 import zarr
 from dask import array as da
 
 from napari.utils._dtype import normalize_dtype
 
+ts = pytest.importorskip('tensorstore')
+torch = pytest.importorskip('torch')
```

Comment:
not related to this PR, but i'm on a slow wifi and couldn't get all this installed easily.